### DISCUSSION TITLE
VO Tools (Breath Removal and Script Validation)

### DISCUSSION THREAD

**Discussion by BrianRollins** [intel/openvino-plugins-ai-audacity#121] [Ideas]
Any chance to put "Breath Removal" in a future release? That's a big time-eater for folks doing narration work in Audacity.

Another suggestion would be something akin to [Pozotron](https://www.pozotron.com/). Where the transcription of the audio is matched up against a text file for accuracy.

Love the tools! Keep up the great work!

**Comment by RyanMetcalfeInt8** [comment]
Thanks @BrianRollins! We'll think about how something like this could be implemented. We usually like to reference implementation on other existing open source models or projects, so if anyone is able to find an open source AI model or implementation of these kinds of features, please send them our way!

**Reply by BrianRollins** [reply]
Let me do some legwork and see if I can find something (least I can do, right?)
